envs/terragrunt.hcl

```diff
@@ -1,14 +1,21 @@
+terraform {
+  source = "${get_repo_root()}//terraform"
+}
+
+# Local values are overwritten by environment specific terragrunt config values
 locals {
-  project_id = get_env("VERTEX_PROJECT_ID")
-  region     = get_env("VERTEX_REGION")
+  env_config  = read_terragrunt_config(find_in_parent_folders("env.hcl"))
+  project_id  = local.env_config.locals.project_id
+  environment = local.env_config.locals.environment
+  region      = get_env("VERTEX_REGION")
```

Comment:
This may just be my ignorance but would we also want to switch the way region is being declared the same as we've done for project_id?